src/readbin/func_start.ml

```diff
@@ -0,0 +1,37 @@
+open Bap.Std
+open Core_kernel.Std
+open Or_error
+
+exception Bad_user_input
+
+let byteweight bin =
+  let tmp = Filename.temp_file "bw_" ".output" in
+  let cmd = Printf.sprintf "bap-byteweight find -x %s > %s" bin tmp in
```

Comment:
this wont work if `bin` contains spaces. Hint quote and escape filenames using `%S` specifier